src/url-query.ts

```diff
@@ -0,0 +1,57 @@
+import queryStringHelpers = require('querystring');
+import urlHelpers = require('url');
```

Comment:
Any reason why you're using `require` for imports here instead of `import * as urlHelpers from url`?

Reply:
Nope, fixed! Thanks.